src/deploy_inst.cc

```diff
@@ -7,7 +7,7 @@ DeployInst::DeployInst(cyclus::Context* ctx)
     : cyclus::Institution(ctx),
       latitude(0.0),
       longitude(0.0),
-      coordinates(latitude, longitude) {}
```

Comment:
I think we need to review/reconfirm the way that we do initialization of variables that are injected.  We don't initialize other variables quite this way, I don't think.